draft-ietf-rats-corim.md

```diff
@@ -617,9 +617,9 @@ Signer map.
 
 ## Signer authority of securely conveyed unsigned CoRIM {#sec-conveyed-signer}
 
-An unsigned (#6.501-tagged) CoRIM may be a payload in an enveloping signed document.
-The CoRIM signer authority is taken from the authenticated credential of the entity that originates the CoRIM.
-A CoRIM role entry that contains the `manifest-signer` role MUST be added to `corim-entity-map`.
+An unsigned (#6.501-tagged) CoRIM may be a payload in an enveloping signed document or conveyed unsigned in the protection scope of a secure channel.
```

Comment:
```suggestion
An unsigned (#6.501-tagged) CoRIM may be a payload in an enveloping signed document, or it may be conveyed unsigned within the protection scope of a secure channel.
```